torchvision/datasets/video_utils.py

```diff
@@ -198,7 +204,13 @@ def subset(self, indices):
         )
 
     @staticmethod
-    def compute_clips_for_video(video_pts, num_frames, step, fps, frame_rate):
+    def compute_clips_for_video(
+        video_pts: torch.Tensor,
+        num_frames: int,
+        step: int,
+        fps: int,
+        frame_rate: Optional[int] = None
+    ) -> Tuple[torch.Tensor, Union[List[slice], torch.Tensor]]:
```

Comment:
Maybe?

```suggestion
    ) -> Tuple[torch.Tensor, Union[List[int], torch.Tensor]]:
```

Reply:
but that would be incorrect?
```python
if step.is_integer():
            # optimization: if step is integer, don't need to perform
            # advanced indexing
            step = int(step)
            return slice(None, None, step)
```
makes it a slice if not a Tensor

Reply:
Now that we differentiate between `_idxs` (nit: should be `_idcs`) and `idxs` that might be true. Let's wait and see if `mypy` is happy.

Reply:
Looks like it's all good! Let me know if you want me to fix the typos (but there are several `idxs` in the files)